### PR TITLE
KOGITO-580 - Task console bootstrap

### PR DESCRIPTION
**Pull request by Sara4994**

### CODE REVIEW

File: packages/task-console/src/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <title>Kogito - Management Console</title>
```
Comment (cristianonicolai) [inline packages/task-console/src/index.html]:
```suggestion
    <title>Kogito - Task Console</title>
```

**Review by cristianonicolai — CHANGES_REQUESTED** [review]
@Sara4994 please  also take a look in the build error

**Comment by cristianonicolai** [comment]
@Sara4994 I wonder if we could reuse some of these config files. I belive most settings will be common anyway so we could share it. It would avoid inconsistencies in the future.

File: Jenkinsfile [hunk 2]
```diff
@@ -29,7 +29,7 @@ pipeline {
                     script {
                         githubscm.checkoutIfExists('kogito-apps', "$CHANGE_AUTHOR", "$CHANGE_BRANCH", 'kiegroup', "$CHANGE_TARGET")
                         wrap([$class: 'Xvnc', takeScreenshot: false, useXauthority: true]) {
-                            sh('yarn run init && yarn build:prod')
+                            sh('yarn run init && yarn build:prod && yarn run lint')
```
Comment (cristianonicolai) [inline Jenkinsfile]: shouldnt lint be part of build:prod instead?

**Review by cristianonicolai — COMMENTED** [review]
@Sara4994 could we also share the webpack files? seems like we can do import and these are not really different between apps, wdyt?

File: Jenkinsfile (hunk 2 above)
Comment (Sara4994) [reply Jenkinsfile]: @cristianonicolai moved lint as common one for both the apps. there is a script in package.json of lerna, which will check lint errors for the both application.

**Review by cristianonicolai — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]

**Pull request merged**